include/oc_helpers.h

```diff
@@ -229,10 +230,14 @@ void oc_concat_strings(oc_string_t *concat, const char *str1, const char *str2);
 #define oc_bool_array_size(ocboolarray) ((ocboolarray).size)
 #define oc_float_array_size(ocfloatarray) ((ocfloatarray).size)
 #define oc_double_array_size(ocdoublearray) ((ocdoublearray).size)
+#define oc_string_array_size(ocstringarray)                                    \
+  ((ocstringarray).size / STRING_ARRAY_ITEM_MAX_LEN)
 #define oc_int_array(ocintarray) (oc_cast(ocintarray, int64_t))
 #define oc_bool_array(ocboolarray) (oc_cast(ocboolarray, bool))
 #define oc_float_array(ocfloatarray) (oc_cast(ocfloatarray, float))
 #define oc_double_array(ocdoublearray) (oc_cast(ocdoublearray, double))
+#define oc_string_array(ocstringarray)                                         \
+  ((char(*)[STRING_ARRAY_ITEM_MAX_LEN])(OC_MMEM_PTR(&(oc_string_array))))
```

Comment:
This is wrong, should be 
```suggestion
  ((char(*)[STRING_ARRAY_ITEM_MAX_LEN])(OC_MMEM_PTR(&(ocstringarray))))
```